fabric_rti_mcp/config/obo_config.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import os
+from dataclasses import dataclass
+from typing import List
+
+
+class OBOFlowEnvVarNames:
+    """Environment variable names for OBO Flow configuration."""
+
+    azure_tenant_id = "AZURE_TENANT_ID"
```

Comment:
can we prefix everything here with `FABRIC_RTI_MCP_` (and also change the class name to `FabricRtiMcpAuthConfig` or something more specific?